tests/Oryx.BuildImage.Tests/PythonSampleAppsTest.cs

```diff
@@ -63,6 +63,44 @@ public void GeneratesScript_AndBuilds()
                 result.GetDebugInfo());
         }
 
+        [Theory]
+        [InlineData(PythonVersions.Python27Version)]
+        [InlineData(PythonVersions.Python36Version)]
+        [InlineData(PythonVersions.Python37Version)]
+        public void GeneratesScript_AndBuilds_Shapely_With_Python(string version)
```

Comment:
Do we actually need this test, given that we're already testing the build on the e2e test?

Reply:
Well, it doesn't harm to have it here, also it's easier to catch any issues here instead of catching in end to end tests